Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm

```diff
@@ -174,13 +174,13 @@
 void RemoteScrollingCoordinator::startDeferringScrollingTestCompletionForNode(WebCore::ScrollingNodeID nodeID, OptionSet<WebCore::WheelEventTestMonitor::DeferReason> reason)
 {
     if (auto monitor = m_page->wheelEventTestMonitor())
-        monitor->deferForReason(reinterpret_cast<WheelEventTestMonitor::ScrollableAreaIdentifier>(nodeID), reason);
+        monitor->deferForReason(reinterpret_cast<WheelEventTestMonitor::ScrollableAreaIdentifier>(nodeID.object().toUInt64()), reason);
```

Comment:
I think we have to fix this so that WheelEventTestMonitor works in terms of ScrollingNodeIDs. Maybe a non-async-scrolling ScrollableArea can still vend a ScrollingNodeID for testing purposes.

Reply:
Yeah I can address this in a follow up patch.